tests/drivers/pwm/pwm_to_gpio_loopback/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pwm/pwm.h>
+#include <zephyr/dt-bindings/gpio/nordic-nrf-gpio.h>
+
+/ {
+	pwm_to_gpio_loopback: pwm_to_gpio_loopback {
+		compatible = "test-pwm-to-gpio-loopback";
+		pwms = <&pwm130 0 0 PWM_POLARITY_NORMAL>;
+		gpios = <&gpio0 1 GPIO_ACTIVE_HIGH>;
+	};
```

Comment:
For sample and test specific devicetree props, `zephyr,user` is used. Bindings files and compatibles are limited mostly to actual hardware :)

```
/ {
        zephyr,user {
                pwms = <&pwm130 0 0 PWM_POLARITY_NORMAL>;
                gpios = <&gpio0 1 GPIO_ACTIVE_HIGH>;
        };
};
```